### ISSUE TITLE
PHP 7 Abstract Syntax Tree

### ISSUE THREAD

**Issue by andreas-bergstrom** [humbug/humbug#165]
With the introduction of the AST in PHP 7, it should be possible to achieve mutation testing without messing with temporary files. Ideas on how to do this?

https://wiki.php.net/rfc/abstract_syntax_tree
https://www.reddit.com/r/PHP/comments/3vxqu0/eli5_new_ast_in_php_7/

**Comment by theofidry** [comment]
This is indeed a feature that is planned for 2.x for which it will be a PHP7.1+ support at least.

**Comment by padraic** [comment]
Closing issue on the basis that it is currently added to 2.0 Roadmap list, and will be discussed there. It's also 99.9% certain to be implemented. I'll allow 0.01% just in case a meteor hits the planet during 2017.
